src/homematicip/class_maps.py

```diff
@@ -36,6 +36,7 @@
     DeviceType.FLOOR_TERMINAL_BLOCK_10: FloorTerminalBlock10,
     DeviceType.FLOOR_TERMINAL_BLOCK_12: FloorTerminalBlock12,
     DeviceType.FLOOR_TERMINAL_BLOCK_6: FloorTerminalBlock6,
+    DeviceType.FLOOR_TERMINAL_BLOCK_8: FloorTerminalBlock8,
```

Comment:
you can assign FLOOR_TERMINAL_BLOCK_8 to class FloorTerminalBlock12. 
Background: in HomeAssistant only the functionalChannel types are important and the changes are getting merged much faster, when only the dependency is updated. A new class would mean code changes in HomeAssistant.